raft/src/test/java/org/apache/kafka/raft/RaftClientTestContext.java

```diff
@@ -113,6 +113,7 @@
         static final int DEFAULT_ELECTION_TIMEOUT_MS = 10000;
 
         private static final TopicPartition METADATA_PARTITION = new TopicPartition("metadata", 0);
+        private static final Uuid METADATA_TOPIC_ID = Uuid.METADATA_TOPIC_ID;
```

Comment:
Do we need to redefine another constant here and in `RaftEventSimulationTest`?

Reply:
good point. Will fix